index.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 // TODO: Use the `URL` global when targeting Node.js 10
-const URLParser = typeof URL === 'undefined' ? require('url').URL : URL;
+const URL = require('url').URL; // eslint-disable-line prefer-destructuring
 
 function testParameter(name, filters) {
 	return filters.some(filter => filter instanceof RegExp ? filter.test(name) : filter === name);
 }
 
 module.exports = (urlString, opts) => {
 	opts = Object.assign({
+		defaultProtocol: 'http:',
```

Comment:
I think it should be specified without the `:`.

Reply:
I'd leave it with the colon. Argument:

```js
> (new URL('https://sindresorhus.com')).protocol
"https:"
```